docs/env.md

```diff
@@ -27,6 +27,7 @@ with a non-standard filesystem layout.
 
 * `FWUPD_MACHINE_KIND` can be used to override the detected machine type, e.g. `physical`, `virtual`, or `container`
 * `FWUPD_HOST_EMULATE` can be used to load test data from `/usr/share/fwupd/host-emulate.d`, e.g. `thinkpad-p1-no-iommu.json.gz`
+* `FWUPD_BACKEND_SAVE` can be used to save fake backend devices to a file. Use `sudo fwupdtool get-devices` to generate.
```

Comment:
I would think it's better to just add a new argument to fwupdtool.

Maybe generate-emulation? And it would then generate as much emulation as possible.

Reply:
> I would think it's better to just add a new argument to fwupdtool.

The tricky bit is that you want to capture the events of things like dumping the firmware or updating the device (that's the interesting bit to emulate, not just the enumeration) -- so it really needs to be a flag to say "save the log for the action" rather than just dumping the coldplugged list. Maybe something like --save-events=PLUGIN-NAME might be better than an env var tho.

Reply:
Yeah I like that better

Reply:
Done; will push when all the CI stuff is fixed.